lib/browser/api/web-contents.ts

```diff
@@ -461,6 +461,11 @@ const addReturnValueToEvent = (event: any) => {
   });
 };
 
+const loggingEnabled = () => {
+  return Object.prototype.hasOwnProperty.call(process.env, 'ELECTRON_ENABLE_LOGGING') ||
+    process.argv.some(arg => arg.toLowerCase().startsWith('--enable-logging'));
```

Comment:
better to use `app.commandLine` to check this sort of thing.

```suggestion
    app.commandLine.hasSwitch('enable-logging')
```

Reply:
https://github.com/electron/electron/pull/25592